### PR TITLE
Ember-cli notes 05 october 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-10/october-5.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-10-05
+
+## Attendees
+
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-10/october-5.md]: Please reply here with agenda items.

Comment (Turbo87) [reply ember-cli/2017-10/october-5.md]: New QUnit testing APIs status

Comment (Turbo87) [reply ember-cli/2017-10/october-5.md]: https://github.com/simplabs/ember-simple-auth/issues/1443#issuecomment-333955394

**Comment by hjdivad** [comment]
cc @rwjblue @kellyselden @turbo87 @trabus

**Pull request merged**